README.md

```diff
@@ -22,15 +22,14 @@ site's theme is based on [Pure Pelican Theme](https://github.com/PurePelicanThem
 
 This will clone the repository on to your system and clone the submodules inside it as well recursively. Additionally, the folder to which it will be cloned to is called `pynepal-website`.
 
-* Set up a virtual env in that folder and activate it
+* Set up a pipenv in that folder and activate it
```

Comment:
Let's not remove the option to setup using virtual env. Some people prefer this way. Just add pipenv as second option.

Reply:
yeah sure.

Reply:
@kaflesudip  I have updated documentation both option .